frontend/lib/norent/letter-builder/confirmation.tsx

```diff
@@ -7,18 +7,41 @@ import {
   USStateChoice,
 } from "../../../../common-data/us-state-choices";
 import { LetterBuilderAccordion } from "./welcome";
+import { getNorentMetadataForUSState } from "./national-metadata";
 
 const checkCircleSvg = require("../../svg/check-circle-solid.svg") as JSX.Element;
 
-const NORENT_FEEDBACK_FORM_URL = "https://airtable.com/shrrnQD3kXUQv1xm3";
+const NATIONAL_LEGAL_AID_URL = "https://www.lawhelp.org";
 const CANCEL_RENT_PETITION_URL = "https://cancelrent.us/";
+const NORENT_FEEDBACK_FORM_URL = "https://airtable.com/shrrnQD3kXUQv1xm3";
 
 export const NorentConfirmation: React.FC<{}> = () => {
   const { session } = useContext(AppContext);
   const letter = session.norentLatestLetter;
   const user = session.norentScaffolding;
-  const stateName =
-    user?.state && getUSStateChoiceLabels()[user?.state as USStateChoice];
+  const state = user?.state && (user?.state as USStateChoice);
+  const stateName = state && getUSStateChoiceLabels()[state];
+
+  // Content from national metadata:
+  const needsDocumentation =
+    state &&
+    getNorentMetadataForUSState(state)?.docs?.isDocumentationALegalRequirement;
```

Comment:
Hmm, it seems like there are _way_ too many `?`'s here... `getNorentMetadataForUSState()` should _always_ return an object that _always_ has a `docs` key on it.  I feel like the situation where `state` is empty is such an aberration (nobody should ever get to this page without being logged in and having a state associated w/ their account) that it will be better in the long run to only show this information if the user has a state, rather than testing everywhere for whether state is empty or not.